css/components/card.css

```diff
@@ -0,0 +1,96 @@
+.card {
+  box-shadow: var(--shadow);
+  background-color: var(--white);
+  border-radius: var(--rounded);
+  max-width: 20rem;
+  display: flex;
+  flex-direction: column;
```

Comment:
put it in the utility class if it used more frequently

Reply:
I have created utility class for that but these lines along with **margin-top:auto** on card footer  helps to keep footer at the bottom always when we use cards with different height in a grid & if anyone uses these card classes, will not have to worry about writing utility class to make it happen. 
Thanks for the review